src/playlist-maker.js

```diff
@@ -1,8 +1,16 @@
 import window from 'global/window';
-import assign from 'object.assign';
 import playItem from './play-item';
 import * as autoadvance from './auto-advance';
 
+// Lightweight Object.assign alternative.
+const assign = (target, source) => {
+  for (let key in source) {
+    if (source.hasOwnProperty(key)) {
+      target[key] = source[key];
+    }
+  }
+};
```

Comment:
I think for a future PR. I'd want to get the generator running babel 6 and include those sorts of things there and then apply those changes here.

This plugin needs to be brought up to speed with the latest version(s) of the generator anyway.